### PR TITLE
Modify rule S2078: Change text to the education framework format [PHP][APPSEC-270]

### PR DESCRIPTION
**Pull request by pedro-oliveira-sonarsource**

### CODE REVIEW

**Comment by pedro-oliveira-sonarsource** [comment]
The validation.adoc text is being modified on this [PR](https://github.com/SonarSource/rspec/pull/1393).

**Comment by pedro-oliveira-sonarsource** [comment]
There is currently a type inference issue with the Ldap component for symfony, reflected with this [ticket](https://sonarsource.atlassian.net/browse/APPSEC-295).

File: rules/S2078/php/metadata.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (egon-okerman-sonarsource) [inline rules/S2078/php/metadata.json:1]: Same as the Python one, I'd remove this unless really necessary to add the CERT standard

**Review by egon-okerman-sonarsource — CHANGES_REQUESTED** [review]

**Review by egon-okerman-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**